conformance/results/pyright/aliases_typealiastype.toml

```diff
@@ -1,4 +1,7 @@
-conformant = "Pass"
+conformant = "Partial"
+notes = """
+incorrectly allows undefined self reference
```

Comment:
Minor nit: Please make notes complete sentences starting with a capital letter and ending with a period (for consistency with all other notes). These appear in the conformance test results page when you mouse over each box in the report, so it's good for them to be consistent.